Licensing requirements for external Yammer users (#647)
MicrosoftDocs/OfficeDocs-O365ITPro: yammer/work-with-external-users/add-external-participants.md

# Add external messaging participants to your Yammer conversations

> Sometimes you need to communicate with outside partners, suppliers, or customers. You can add external participants to conversations in Yammer so that you can work with the people you need, even if they aren't in your Yammer network. For example, you can add external participants to a discussion or use instant messaging to quickly get a response. External participants can view and download files that have been uploaded to the conversation, and upload files. You'll be able to see when a conversation includes an external participant, and you can remove an external participant when you need to. If the external user is belong an Office 365 Tenant, a Yammer enterpise license is needed for the user. If the external user is not belong a Office 365 Tenant, the user must have a valid work or school email account to participate as an external Yammer user, on this case the email account can not be a public acconut (@hotmail.com, @outlook.com, @gmail.com or @yahoo.com) 

Reviewer: ```suggestion
Sometimes you need to communicate with outside partners, suppliers, or customers. You can add external participants to conversations in Yammer so that you can work with the people you need to, even if they aren't in your Yammer network. For example, you can add external participants to a discussion or use instant messaging to quickly get a response. External participants can view and download files that have been uploaded to the conversation, and upload files. You'll be able to see when a conversation includes an external participant, and you can remove an external participant when you need to. If the external user belongs to an Office 365 Tenant, a Yammer Enterpise license is needed for the user. If the external user does not not belong to an Office 365 Tenant, the user must have a valid work or school email account to participate as an external Yammer user. In this case, the email account may not be a public account (@hotmail.com, @outlook.com, @gmail.com, or @yahoo.com) 
```
